session.go

```diff
@@ -273,12 +281,14 @@ func (session *Session) ReceiveWith(localClientID, remoteClientID string, buf []
 
 		if CompareSeq(packet.SequenceId, session.recvWindowStartSeq) >= 0 {
 			if _, ok := session.recvWindowData[packet.SequenceId]; !ok {
-				if session.recvWindowUsed+uint32(len(packet.Data)) > session.recvWindowSize {
+				if session.recvWindowUsed+uint32(len(packet.Data)) > session.recvWindowSize &&
+					packet.SequenceId != session.recvWindowStartSeq {
```

Comment:
Why? It's caller's responsibility to make sure window size > mtu

Reply:
Removed